### PR TITLE
Changed keystore and truststore title to be more searchable

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters

Changed Java keystore and truststore title to add keystore and truststore words to make it more searchable

### CODE REVIEW

File: docs/products/kafka/howto/keystore-truststore.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Configure Java SSL to access Apache Kafka®
-=================================================
+Configure Java SSL keystore and trustore to access Apache Kafka®
+================================================================
```
Comment (RyanSkraba) [inline docs/products/kafka/howto/keystore-truststore.rst:2]:
```suggestion
Configure Java SSL keystore and truststore to access Apache Kafka®
==================================================================
```

File: _toc.yml [hunk 2]
```diff
@@ -217,6 +217,7 @@ entries:
             title: Security
             entries:
               - file: docs/products/kafka/howto/keystore-truststore
+                title: Configure Java SSL keystore and trustore to access Apache Kafka
```
Comment (RyanSkraba) [inline _toc.yml:220]:
```suggestion
                title: Configure Java SSL keystore and truststore to access Apache Kafka
```

**Review by RyanSkraba — APPROVED** [review]
After fixing the typo, this LGTM !

**Comment by ftisiot** [comment]
Dunno how, I managed to add 5 words and spell 2 of them wrong! 
Thanks @RyanSkraba, if you're happy with the final result, feel free to merge the PR!

**Pull request merged**